### ISSUE TITLE
[BUG] - Accordion not accessible properly with VoiceOver or JAWS

### ISSUE THREAD

**Issue by outputboy** [nextui-org/nextui#3194]
### NextUI Version

2.4.0

### Describe the bug

The accordion is accessible with keyboard and Mac voiceover. 
However, when accessing via iPhone voiceover or JAWS, we have to tab space/enter 1 extra time to expand the accordion. 
Seems there is an extra layer of trigger when using JAWS, not sure why.

### Your Example Website or App

https://nextui.org/docs/components/accordion

### Steps to Reproduce the Bug or Issue

1. Activate Voiceover on iphone.
2. Open https://nextui.org/docs/components/accordion 
3. Try expand one accordion
4. We have to click screen 3 times to expand it. Normally it should be 2 tabs

### Expected behavior

As a user using Voiceover on iPhone, I expect to click twice on accordion title to expand the accordion.

### Operating System Version

IOS 17.5.1 iPhone13

### Browser

Chrome
